eng/pipelines/templates/steps/cmake-build.yml

```diff
@@ -16,4 +16,8 @@ steps:
 
   - script: cmake --build . --config ${{ parameters.BuildType }}
     workingDirectory: build
-    displayName: cmake build
\ No newline at end of file
+    displayName: cmake build
+
+  - script: git status --ignored --untracked=all --show-ignored-directory
+    workingDirectory: .
+    displayName: git status
```

Comment:
new line needed?

Reply:
Yes, but this script change will be reverted. Testing out some stuff in CI machines.

I have another PR out that fixes all these lack of new lines in the repo:
https://github.com/Azure/azure-sdk-for-c/pull/1242